build.sbt

```diff
@@ -75,10 +75,10 @@ lazy val sharedSettings = Seq(
   ),
 
   scalacOptions ++= Seq(
-    "-unchecked", "-deprecation", "-feature", "-Xlint",
-    "-Ywarn-adapted-args", "-Ywarn-dead-code", "-Ywarn-inaccessible",
-    "-Ywarn-nullary-override", "-Ywarn-nullary-unit",
-    "-Xlog-free-terms"
+    "-unchecked", "-deprecation", "-feature", // "-Xlint",
+    // "-Ywarn-adapted-args", "-Ywarn-dead-code", "-Ywarn-inaccessible",
+    // "-Ywarn-nullary-override", "-Ywarn-nullary-unit",
+    // "-Xlog-free-terms"
```

Comment:
I would keep these options and override the options in dottySettings. I am trying to minimise the diff with the the upstream repository to keep rebasing as simple as possible

Reply:
will remove the comments